src/Cooker/providers/GitProvider/getHashListFromHash.js

```diff
@@ -8,26 +8,30 @@ function getHashListFromHashToCommit(sha, commit) {
   }
 
   return new Promise((resolve, reject) => {
-    let hitPreviousRelease = false
+    let reachedHash = false
     const list = []
     commit
       .history()
       .on('commit', commit => {
-        if (hitPreviousRelease) {
+        if (reachedHash) {
           return
         }
 
         if (commit.sha() === sha) {
-          hitPreviousRelease = true
+          reachedHash = true
         } else {
           list.unshift(commit.sha())
         }
       })
       .on('end', () => {
-        if (hitPreviousRelease || list.length < 50) {
+        if (reachedHash || sha === 'Big Bang') {
```

Comment:
Is this "Big Bang" a placeholder name for when there is no previous release tag? It is just internal name?